test/unit/modunit_pico_mld.c

```diff
@@ -118,16 +119,12 @@ END_TEST
 START_TEST(tc_pico_mld_report_expired)
 {
     struct mld_timer *t = PICO_ZALLOC(sizeof(struct mld_timer));
-    struct pico_ip6 zero = {{0}};
-    struct mcast_parameters p;
+    struct pico_ip6 zero = {0};
```

Comment:
Apparently, this is a known GCC bug:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119
But it's status is set to FIXED so not sure what's up.
Still, I'd like to build with your GCC version (if it is different than mine) and see if I get the warnings.
If I get the warnings, maybe we should agree on a GCC version to build with or the version of GCC used on Jenkins should be explicitly stated somewhere.